enterprise/app/services/sla/evaluate_applied_sla_service.rb

```diff
@@ -25,8 +49,21 @@ def still_within_threshold?(threshold)
     Time.zone.now.to_i < threshold
   end
 
+  def calculate_threshold(start_time, threshold_seconds)
+    if @business_hours_configured
+      if start_time.in_working_hours?
+        start_time.to_i + threshold_seconds.working.seconds
```

Comment:
```
17:46:48 worker.1   | /enterprise/app/services/sla/evaluate_applied_sla_service.rb:57:in `+'
17:46:48 worker.1   | /enterprise/app/services/sla/evaluate_applied_sla_service.rb:57:in `calculate_threshold'
17:46:48 worker.1   | /enterprise/app/services/sla/evaluate_applied_sla_service.rb:67:in `check_first_response_time_threshold'
17:46:48 worker.1   | /enterprise/app/services/sla/evaluate_applied_sla_service.rb:45:in `block in check_sla_thresholds'
```
maybe need to add `.value`
`start_time.to_i + threshold_seconds.working.seconds.value`